planner-agent/src/workflows/vibeTodoCreator/workflow.py

```
@@ -269,57 +284,99 @@
+                                    uuid=task_data.get("uuid"),
+                                    bountyId=self.context["bounty_id"],
+                                    bountyType=self.bounty_type,
+                                    issueUuid=issue["uuid"],
+                                )
+                                task_models.append(task_model)
+                        except Exception as e:
+                            log_error(
+                                e,
+                                f"Failed to process task {task_data.get('info', 'unknown')} "
+                                f"with UUID {task_data.get('uuid', 'unknown')}",
                             )
```

Pylint (3.3.7)

[refactor] Too many nested blocks (7/5)

(R1702)